packages/cletus/src/operations/dba.tsx

```diff
@@ -736,4 +933,171 @@ const OPERATOR_MAP: Record<string, string> = {
   after: "≥",
   oneOf: " ONE OF ",
   isEmpty: "IS EMPTY",
-};
\ No newline at end of file
+};
+
+/**
+ * Build a Zod schema for a single record based on type definition
+ */
+function buildRecordSchema(type: TypeDefinition) {
```

Comment:
We already have this defined in architect.ts export that function and use it and remove this